skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -1360,41 +1360,6 @@ def test_multiindex_fancy_indexing_incomplete_label(self):
                                                                'd']},
                                     index=[0, 1]))
 
-    def test_multiindex_complicated_axis(self):
-        a = RNA("UUAG", metadata={0: 0}, positional_metadata={0: [1, 2, 3, 4]})
-        b = RNA("UAAG", metadata={1: 0}, positional_metadata={1: [1, 2, 3, 4]})
-        c = RNA("UAA-", metadata={2: 0}, positional_metadata={2: [1, 2, 3, 4]})
-        d = RNA("UA-G", metadata={3: 0}, positional_metadata={3: [1, 2, 3, 4]})
-        msa = TabularMSA([a, b, c, d], metadata={'x': 'y'},
-                         positional_metadata={'c': ['a', 'b', 'c', 'd']},
-                         index=[('a', 'x', 0), ('a', 'x', 1), ('a', 'y', 2),
-                                ('b', 'x', 0)])
-
-        self.assertEqual(self.get(msa, (([False, True, False, True],
-                                         'x', 0), Ellipsis)),
-                         TabularMSA([d], metadata={'x': 'y'},
-                                    positional_metadata={'c': ['a', 'b', 'c',
-                                                               'd']},
-                                    index=[('b', 'x', 0)]))
-
-    @unittest.skipIf(tuple(map(int, pd.__version__.split('.'))) < (1, 2, 5),
-                     "Old pandas will return empty frame")
-    def test_multiindex_complicated_axis_empty_selection(self):
-        a = RNA("UUAG", metadata={0: 0}, positional_metadata={0: [1, 2, 3, 4]})
-        b = RNA("UAAG", metadata={1: 0}, positional_metadata={1: [1, 2, 3, 4]})
-        c = RNA("UAA-", metadata={2: 0}, positional_metadata={2: [1, 2, 3, 4]})
-        d = RNA("UA-G", metadata={3: 0}, positional_metadata={3: [1, 2, 3, 4]})
-        msa = TabularMSA([a, b, c, d], metadata={'x': 'y'},
-                         positional_metadata={'c': ['a', 'b', 'c', 'd']},
-                         index=[('a', 'x', 0), ('a', 'x', 1), ('a', 'y', 2),
-                                ('b', 'x', 0)])
-        # Pandas will KeyError when the intersection is empty
-        # change appears to have happened in:
-        # https://github.com/pandas-dev/pandas/pull/42245
-        # but this was not bisected to confirm
-        with self.assertRaises(KeyError):
-            self.get(msa, (([False, True, False, True], 'x', 2), Ellipsis))
```

Comment:
@ebolyen, you okay with this?

Reply:
Sorry, did not see this! Yeah that's totally fine, these were observational tests more than good features.

Reply:
Wonderful, thank you @ebolyen!